src/scrolling.ts

```diff
@@ -9,21 +10,12 @@ let verticallyScrolling = new Map()
 
 let opts = { smooth: null, duration: null }
 async function getSmooth() {
-    if (opts.smooth === null)
-        opts.smooth = await Native.getConfElsePrefElseDefault(
-            "smoothscroll",
-            "general.smoothScroll",
-            "false",
-        )
+    if (opts.smooth === null) opts.smooth = await config.get("smoothscroll")
```

Comment:
config.get isn't async :)

Reply:
You're right, thank you! It'd be cool if Typescript told you about unnecessary `await`s.